lib/chef/provider/service/upstart.rb

```diff
@@ -62,7 +62,7 @@ def initialize(new_resource, run_context)
           end
 
           platform, version = Chef::Platform.find_platform_and_version(run_context.node)
-          if platform == "ubuntu" && (8.04..9.04).include?(version.to_f)
+          if platform == "ubuntu" && (8.04..9.04).cover?(version.to_f)
```

Comment:
wow, this one is a lot faster:

https://github.com/JuanitoFatas/fast-ruby#range

although for Float, i'd think `#include?` would have to be an alias to `#cover?`